PROCESSORS.md

```diff
@@ -50,7 +50,9 @@
 - [PutS3Object](#puts3object)
 - [PutSFTP](#putsftp)
 - [PutSQL](#putsql)
+- [PutSplunkHTTP](#putsplunkhttp)
```

Comment:
nitpick: this might not be in alphabetical order

Reply:
Reordered it in https://github.com/apache/nifi-minifi-cpp/pull/1219/commits/412fa041455d782eedc52bdfb24c680ae7e56808#diff-fd2410931e7fdc4bf8b3ce23f5f7a27c7aacdf9337320626d86d806448c90b9bL52